src/main/java/org/broadinstitute/hellbender/engine/spark/GATKRegistrator.java

```diff
@@ -93,5 +94,6 @@ private void registerGATKClasses(Kryo kryo) {
         kryo.register(ReadsKey.class, new FieldSerializer(kryo, ReadsKey.class));
         kryo.register(ReadsKey.KeyForFragment.class, new FieldSerializer(kryo, ReadsKey.KeyForFragment.class));
         kryo.register(ReadsKey.KeyForPair.class, new FieldSerializer(kryo, ReadsKey.KeyForPair.class));
+        kryo.register(SerializableOpticalDuplicatesFinder.class, new FieldSerializer(kryo, SerializableOpticalDuplicatesFinder.class));
```

Comment:
Weren't we getting rid of this class?  Should this be replaces with a registration of OpticalDuplicateFinder?

Reply:
yes